subprojects/dependency-management/src/main/java/org/gradle/internal/locking/DefaultDependencyLockingProvider.java

```diff
@@ -80,7 +84,13 @@ public DependencyLockingState loadLockState(String configurationName) {
     @Override
     public void persistResolvedDependencies(String configurationName, Set<ModuleComponentIdentifier> resolvedModules) {
         if (writeLocks) {
-            lockFileReaderWriter.writeLockFile(configurationName, getModulesOrdered(resolvedModules));
+            List<String> modulesOrdered = getModulesOrdered(resolvedModules);
+            lockFileReaderWriter.writeLockFile(configurationName, modulesOrdered);
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("Persisted dependency lock state for configuration '{}', state is: {}", configurationName, modulesOrdered);
+            } else {
+                LOGGER.lifecycle("Persisted dependency lock state for configuration '{}'", configurationName);
```

Comment:
The one annoying thing is that this statement will appear in the middle of whatever task output that caused the configuration to be resolved.

Reply:
I think it's fine, and expected.